changelog/_unreleased/2021-04-15-ignored_url_parameters for cache.md

```diff
@@ -0,0 +1,46 @@
+---
+title:          Ignore some HTTP Parameters from Cache Key generation  
+author:         Björn Herzke  
+issue:          NEXT-14806  
+author_github:  @wrongspot  
+___
+# Storefront
+* Added new Class `Shopware\Storefront\Framework\Cache\CacheKeyProvider`
+* Added configuration `httpcache.ignored_url_parameters` in `storefront.yaml` to blacklist parameters from Cache Key generation
+
+### Usage of Parameter Blacklist
```

Comment:
This should be into `Upgrade Information`. Root headlines only bundle names and Upgrade Information